Tests/L0/Gradle/_suite.ts

```diff
@@ -82,6 +82,10 @@ function assertCodeAnalysisBuildSummaryContains(stagingDir: string, expectedStri
     assertBuildSummaryContains(path.join(stagingDir, '.codeAnalysis', 'CodeAnalysisBuildSummary.md'), expectedString);
 }
 
+function assertCodeAnalysisBuildSummaryDoesNotContain(stagingDir: string, string: string): void {
```

Comment:
string is a reserved word in TS so it would be better to avoid using as a variable name as it creates confusion